### PR TITLE
update certificate design

### PR DESCRIPTION
**Pull request by emraan004**

### CODE REVIEW

File: openedx/features/student_certificates/helpers.py [hunk 1]
```diff
@@ -156,3 +157,42 @@ def get_pdfkit_html(image_base64):
     :return: html
     """
     return PDFKIT_HTML_STRING.format(image_tag=PDFKIT_IMAGE_TAG.format(base64_img=image_base64))
+
+
+def override_update_certificate_context(request, context, course, user_certificate):
+    """
+    This method adds custom context to the certificate
+    :param request:
```
Comment (saad-waqar) [inline openedx/features/student_certificates/helpers.py]: lets either remove the param descriptions or add them

File: openedx/features/student_certificates/helpers.py [hunk 2]
```diff
@@ -156,3 +157,42 @@ def get_pdfkit_html(image_base64):
...
+    else:
+        context['border_class'] = ''
+
+    context['download_pdf'] = reverse('download_certificate_pdf',
+                                      kwargs={'certificate_uuid': user_certificate.verify_uuid})
+    context['verification_url'] = get_certificate_verification_url(user_certificate)
+    context['social_sharing_urls'] = get_philu_certificate_social_context(course, user_certificate)
+
+
+def get_certificate_verification_url(certificate):
```
Comment (saad-waqar) [inline openedx/features/student_certificates/helpers.py]:
```
key = 'PREVIEW_VERIFICATION'

if certificate.pk:
    key = certificate.certificate_verification_key.verification_key

return settings.LMS_ROOT_URL + reverse('certificate_verification', kwargs={'key':  key})
```

File: openedx/features/student_certificates/helpers.py [hunk 3]
```diff
@@ -156,3 +157,42 @@ def get_pdfkit_html(image_base64):
...
+    """
+    This method generates a verification url for course certificate
+    :param certificate:
```
Comment (saad-waqar) [inline openedx/features/student_certificates/helpers.py]: param description here

File: openedx/features/student_certificates/helpers.py (hunk 2 above)
Comment (saad-waqar) [inline openedx/features/student_certificates/helpers.py]: we can make this a method of the model maybe?

File: openedx/features/student_certificates/helpers.py (hunk 3 above)
Comment (zeshanarifios) [inline openedx/features/student_certificates/helpers.py]: param detail missing

**Pull request merged**